include/inset_state.hpp

```diff
@@ -202,7 +230,7 @@ class InsetState
   void replace_target_area(const std::string &, double);
   void rescale_map(unsigned int, bool);
   void revert_smyth_craster_projection();
-  void rings_are_simple();
+  void rings_are_simple() const;
```

Comment:
I feel this method might have become redundant; as the improved `is_simple` already does everything it does.

Reply:
Actually you are correct. I will remove it and replace it where used.